goToExercise.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+function print_usage_and_exit {
+  echo "Usage: ./goToExercise.sh <number>"
+  exit 1
+}
+
+EX="$1"
+
+if [ -z  "$EX" ]
+then
+  print_usage_and_exit
+else
+  git add .
+  git commit -m "Some work"
```

Comment:
maybe message could be an optional argument to the script?

Reply:
I don't think that's necessary, I'd leave it as it is :)